### DISCUSSION TITLE
Ploting on main pc but farming on second

### DISCUSSION THREAD

**Discussion by BartoszN1990** [Chia-Network/chia-blockchain#2474] [Q&A]
Hello i just try resolved a issu with start farming. Well because i have righ good pc i decided to start ploting on my Main pc but syns it be restarted or turn on u must move my farm to ex Laptop or other pc.

And my question is if i use windows Main PC for plots aready have 25 plots.
I try to move to laptop with ubuntu but i cant sort out how to sort out usb drivers and set up so i install windows and coppy full walet to laptop from main pc . i start walelt and add plots.

On laptop wallet show syns but 0 connection plots included show farming.
On main pc show syns but 0 connection i remove plots.

And my question is did i should turn on wallet or somethins on main pc when i ploting ? to dont disconect wallet from laptop ? or i can use 2 devices on 1 wallet the same time dont losing syns and farming signal ? 

Please give any advice how to sort out this at the moment w devices show syns 0 connection so i udnerstound my wallets syns but not farming so my plots dont  working right ?

**Comment by dorkmo** [comment]
first make sure in your network's router that the port 8444 is forwarded to your farming pc. you'll also want to make sure that you added the directory where you moved the plots on the laptop. you dont need to run the wallet on both. you only need to run the node on the windows plotter if you want to farm the plots you havent moved yet. best of luck!

**Comment by BartoszN1990** [comment]
Well syns fiew days i have that problem my walelt dont have connection i try restart reisntal and still the same about the forwarded port i dont know how to do that on my router ;/ i have to many option i d ont know what i should write where
